src/resources/views/components/forms/signup.blade.php

```diff
@@ -1,72 +1,56 @@
-<!DOCTYPE html>
-<html lang="en">
+    @extends('layouts.app')
 
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    {{-- <link rel="stylesheet" href="{{ url('assets/css/registration.css') }}"> --}}
-    <title>Sign Up</title>
-    @vite(['resources/css/app.css', 'resources/scss/app.scss', 'resources/js/app.js'])
-</head>
+@section('title', 'Sign Up')
 
-<body>
-       <div class="container">
+@section('content')
+    <div class="form-container">
         <div class="title">
             <h3>Sign Up</h3>
         </div>
 
-        <form action="/register" method="POST">
+        <form action="/register" method="POST" id="signup-form">
             @csrf
             <div class="user-details">
 
-
                 <div class="input-box">
-                    <input type="email" name="email" placeholder="example@gmail.com" autocomplete="off">
-                    <span>Email address</span>
+                    <input type="email" name="email" placeholder="example@gmail.com" autocomplete="off" required>
```

Comment:
Since you've already handled the requests in SigninRequest and SgnupRequest, adding HTML5 validations is not necessary. You can add frontend validations in the future.